auxiliary/ansi-aux.lsp

```diff
@@ -1024,10 +1024,10 @@ the condition to go uncaught if it cannot be classified."
   "Create a test that FORM, which should produce a fresh value,
    does not improperly introduce sharing during constant folding."
   `(deftest ,name
-     (flet ((%f () (declare (optimize (speed 3) (safety 0) (space 0)
```

Comment:
Should work when loading from `eus-test.l`

https://github.com/euslisp/ansi-test/blob/gsoc2018/eus-test.l#L102